scripts/push-docker-images

```diff
@@ -60,18 +60,21 @@ if [[ $MANIFEST == "true" ]]; then
     echo '{"experimental":"enabled"}' > $DOCKER_CLI_CONFIG
     echo "Created docker config file"
   fi
-  cat <<< $(jq '.+{"experimental":"enabled"}' $DOCKER_CLI_CONFIG) > $DOCKER_CLI_CONFIG
+  cat <<< "$(jq '.+{"experimental":"enabled"}' $DOCKER_CLI_CONFIG)" > $DOCKER_CLI_CONFIG
   echo "Enabled experimental CLI features to execute docker manifest commands"
   manifest_exists=$(docker manifest inspect $IMAGE_REPO:$VERSION > /dev/null ; echo $?)
   if [[ manifest_exists -eq 0 ]]; then
     echo "manifest already exists"
-    EXISTING_IMAGES=($(docker manifest inspect $IMAGE_REPO:$VERSION | jq -r '.manifests[] | "\(.platform.os)-\(.platform.architecture)"'))
+    EXISTING_IMAGES=()
+    while IFS='' read -r line; do 
+      shell_files+=("$line"); 
```

Comment:
Can you explain this a bit? Not sure why it needs to be here

Reply:
it was recommended here: https://github.com/koalaman/shellcheck/wiki/SC2207

It basically ensures that the array additions are added as separate elements without splitting the splitting globs mid-line.

Reply:
oh or copy/paste error on the var? prob should be `EXISTING_IMAGES` right?

Reply:
GOOD CATCH